programs/svm-spoke/src/utils/message_utils.rs

```diff
@@ -88,7 +88,6 @@ pub fn invoke_handler<'info>(
 
     let instruction = Instruction { program_id: message.handler, accounts, data };
 
-    // TODO: consider if the message handler requires signed invocation.
```

Comment:
Open question resolved: no need for the handler to authenticate the caller, as we don’t handle permissioned transactions in EVM (cc: @Reinis-FRP )